src/apps/miscapps/h3ToGeoHier.c

```diff
@@ -95,55 +97,92 @@ void recursiveH3IndexToGeo(H3Index h, int res, int isKmlOut) {
     }
 }
 
-int main(int argc, char* argv[]) {
-    // check command line args
-    if (argc < 2 || argc > 5) {
-        fprintf(stderr, "usage: %s H3Index [resolution outputMode]\n", argv[0]);
-        exit(1);
+int main(int argc, char *argv[]) {
+    int res = 0;
+    H3Index parentIndex = 0;
+    char userKmlName[BUFF_SIZE] = {0};
+    char userKmlDesc[BUFF_SIZE] = {0};
+
+    Arg helpArg = {.names = {"-h", "--help"},
+                   .helpText = "Show this help message."};
+    Arg resArg = {.names = {"-r", "--resolution"},
+                  .scanFormat = "%d",
+                  .valueName = "res",
+                  .value = &res,
+                  .helpText =
+                      "Resolution, if less than the resolution of the parent "
+                      "only the parent is printed. Default 0."};
```

Comment:
Nit: `Default 0` seems incorrect; the default is the res of the parent.

Reply:
This program accepts coarser resolutions as valid input, so this is documenting that existing behavior.  From that perspective, 0 does seem to really be the default. I changed it to be a little clearer.